common/test/acceptance/tests/studio/test_studio_course.py

```diff
@@ -0,0 +1,436 @@
+        self._assert_is_admin(other)
+
+        current = self.page.get_user(self.user.get('email'))
+        self.assertTrue(current.can_demote)
+        self.assertTrue(current.can_delete)
+        current.click_delete()
+
+        self.log_in(self.user)
+        self._assert_current_course(visible=False)
+
+    def test_admins_should_be_able_to_give_course_ownership_to_someone_else(self):
+        """
+        Scenario: Admins should be able to give course ownership to someone else
+        Given I have opened a new course in Studio
+        And I am viewing the course team settings
+        When I add other user to the course team
+        And I make other user a course team admin
+        And I remove admin rights from myself
+        And other user logs in
+        And he selects the new course
+        And he views the course team settings
+        And he deletes me from the course team
+        And I am logged into studio
+        Then I do not see the course on my page
+        """
+        self.page.add_user_to_course(self.other_user.get('email'))
+        self._assert_user_present(self.other_user, present=True)
+
+        current = self.page.get_user(self.user.get('email'))
+        self.assertFalse(current.can_demote)
+        self.assertFalse(current.can_delete)
+        self.assertIn("Promote another member to Admin to remove your admin rights", current.no_change_warning_text)
+
+        other = self.page.get_user(self.other_user.get('email'))
+        other.click_promote()
+        self._expect_refresh()
+        other = self.page.get_user(self.other_user.get('email'))
+        self._assert_is_admin(other)
+
+        current = self.page.get_user(self.user.get('email'))
+        self.assertTrue(current.can_demote)
+        self.assertTrue(current.can_delete)
+        current.click_demote()
+        self._expect_refresh()
+        current = self.page.get_user(self.user.get('email'))
+        self._assert_is_staff(current, can_manage=False)
+
+        self.log_in(self.other_user)
+        self._go_to_course_team_page()
+
+        current = self.page.get_user(self.user.get('email'))
+        current.click_delete()
+        self._expect_refresh()
+        self._assert_user_present(self.user, present=False)
+
+        self.log_in(self.user)
+        self._assert_current_course(visible=False)
+
+    def test_admins_should_be_able_to_remove_their_own_admin_rights(self):
+        """
+        Scenario: Admins should be able to remove their own admin rights
+        Given I have opened a new course in Studio
+        And I am viewing the course team settings
+        Then I should be marked as an admin
+        And I can manage users
+        When I remove admin rights from myself
+        Then I should not be marked as an admin
+        And I cannot manage users
+        And I cannot make myself a course team admin
+        """
+        self.page.add_user_to_course(self.other_user.get('email'))
```

Comment:
Thank you for converting these Lettuce tests! I imagine you converted exactly what was there, but I'm finding the amount duplication between the tests to be a bit extreme. For instance, this test can easily be folded into the previous test by simply adding 

self._assert_can_not_manage_users()
self.assertFalse(current.can_promote)

after line 388. While I see the value in having independent tests for certain things, we also are constantly battling to make build times faster. Therefore I think it is worth losing the duplicative independent test cases in this case.

I would also delete test_admins_can_add_other_users, since the steps in it are the exact same first steps in test_added_users_cannot_add_or_delete_other_users.